### ISSUE TITLE
Error: unsupported CUDA toolkit 11.5

### ISSUE THREAD

**Issue by EricLBuehler** [coreylowman/cudarc#245]
When building cudarc, a user reported the following error:

```
error: failed to run custom build command for cudarc v0.11.4

Caused by:
process didn't exit successfully: /root/mistral.rs/target/release/build/cudarc-7276cb883fd7b801/build-script-build (exit status: 101)
--- stdout
cargo:rerun-if-changed=build.rs
cargo:rerun-if-env-changed=CUDA_ROOT
cargo:rerun-if-env-changed=CUDA_PATH
cargo:rerun-if-env-changed=CUDA_TOOLKIT_ROOT_DIR

--- stderr
thread 'main' panicked at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cudarc-0.11.4/build.rs:73:14:
Unsupported cuda toolkit version: 11.5. Please raise a github issue.
note: run with RUST_BACKTRACE=1 environment variable to display a backtrace
```

Is this a feature to be a added to `cudarc`? Refs EricLBuehler/mistral.rs#396.

**Comment by coreylowman** [comment]
Yes we can add 11.5/11.6

**Comment by EricLBuehler** [comment]
Thank you! Would you please write a release for this?

**Comment by coreylowman** [comment]
@EricLBuehler just released under cudarc 0.11.5

**Comment by EricLBuehler** [comment]
@coreylowman great, thank you for being so quick!
